### PR TITLE
John Scudder's review

### PR DESCRIPTION
**Pull request by marco-tiloca-sics**
This PR addresses the review from John Scudder archived at https://mailarchive.ietf.org/arch/msg/core/CaK5R4gq1wc3CN5-dTN3uRXoU2U/

### CODE REVIEW

File: draft-ietf-core-oscore-edhoc.md [hunk 1]
```diff
@@ -552,7 +553,9 @@ For all entries, the Change Controller is IETF, and the reference is \[RFC-XXXX]
 
 IANA is requested to create a new "EDHOC Authentication Credential Types" registry within the "Ephemeral Diffie-Hellman Over COSE (EDHOC)" registry group defined in {{I-D.ietf-lake-edhoc}}.
 
-The registry uses the "Expert Review" registration procedure {{RFC8126}}. Expert Review guidelines are provided in {{review}}.
+As registration procedure, the registry uses either "Standards Action with Expert Review", or "Specification Required" per {{Section 4.6 of RFC8126}}. Expert Review guidelines are provided in {{review}}.
```
Comment (cabo) [inline draft-ietf-core-oscore-edhoc.md]: Is that a "registration procedure"?  I'd say registration policy.

File: draft-ietf-core-oscore-edhoc.md [hunk 2]
```diff
@@ -577,7 +580,7 @@ Initial entries in this registry are as listed in {{pre-reg}}.
 
 ## Expert Review Instructions ## {#review}
 
-The IANA registry established in this document is defined as "Expert Review". This section gives some general guidelines for what the experts should be looking for; but they are being designated as experts for a reason, so they should be given substantial latitude.
+The IANA registry established in this document is defined as "Standards Action with Expert Review" or "Specification Required". This section gives some general guidelines for what the experts should be looking for; but they are being designated as experts for a reason, so they should be given substantial latitude.
```
Comment (cabo) [inline draft-ietf-core-oscore-edhoc.md]: Is there a condition on that "or"?  
(I.e., who gets to choose?)

File: draft-ietf-core-oscore-edhoc.md (hunk 1 above)
Comment (marco-tiloca-sics) [reply draft-ietf-core-oscore-edhoc.md]: Fixed in https://github.com/core-wg/oscore-edhoc/commit/6d45a8c58598679157ee0ff6ced4cca0825e788c

File: draft-ietf-core-oscore-edhoc.md (hunk 2 above)
Comment (marco-tiloca-sics) [reply draft-ietf-core-oscore-edhoc.md]: Fixed in https://github.com/core-wg/oscore-edhoc/commit/9f1cbd4e511ebfe91b9105fac08609d724b92a35

**Review by cabo — APPROVED** [review]

**Pull request merged**